### PR TITLE
Mitigating responsive in web

### PR DESCRIPTION
**Pull request by duehoa1211**
- [x] Mitigating collection screen responsive
           1.  Overview tab ✅ 
           2. Episode tab ✅ 
           3.  Credit tab ✅ 
           4.  Images tab ✅ 
           5.  Tags tab ✅ 
           6. Files tab ✅ 
- [x] Mitigating utilities screen reponsive

### CODE REVIEW

**Comment by duehoa1211** [comment]
Link to #745

**Review by hidden4003 — APPROVED** [review]

**Comment by ElementalCrisis** [comment]
Your changes created new issues for the options bar at lower res now. 

image
image
image

Review the mockups in Figma (check Components page) as at 1280 and below, the search bar is always shown and takes up the entire width.

The button and padding styling has been changed as well. 

Correct
image

Your Version
image

**Comment by ElementalCrisis** [comment]
Here's an example when the options bar still fits at a lower browser res.
https://github.com/ShokoAnime/Shoko-WebUI/assets/9443295/4b551811-17bc-43b0-9e6f-fbeadb55cb7c

Example when it does not. 
https://github.com/ShokoAnime/Shoko-WebUI/assets/9443295/de701896-f39d-423b-8f6c-e4d15bd1a0ca

File: src/components/Input/Input.tsx [hunk 1]
```diff
@@ -1,9 +1,13 @@
-import React, { useContext, useEffect, useRef } from 'react';
+import React, { useContext, useEffect, useMemo, useRef } from 'react';
+import { mdiCloseCircleOutline } from '@mdi/js';
 import { Icon } from '@mdi/react';
 import cx from 'classnames';
+import { useEventCallback } from 'usehooks-ts';
```
Comment (harshithmohan) [inline src/components/Input/Input.tsx:5]: Use our custom `useEventCallback` hook.
At other places too

**Review by harshithmohan — CHANGES_REQUESTED** [review]

**Pull request merged**